news/2024/2024-10-09-resurrection-cup-2024-results.md

```diff
@@ -0,0 +1,417 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8266808">[Boy]DaLat</a></p>
+
+I would say transitioning from osu!(lazer) to custom overlay might need a little bit of work. Each overlay has its own set of defined workflows, comparing to the standardized flow of osu!(lazer). Streamers, to some extent, need to be introduced to a different workflow, user experience, etc, which might take a while to adapt to. At first, developing overlays are pretty much trivial, you read the state from tosumemory, compare changes, update the state on the UI. But when I started to scale things up, for example, when I have to add more custom state, more interactions, I need to change my way of thinking over and over again. Because things might break miserably if I do not engineering thoroughly. That is why when I was asked to add feature abcdxyz to the overlay, during the production of the tournament, I got really upset because now I had to expand the structure of the overlay in such a short amount of time to keep up with the matches.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/12840110" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Phreel.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/12840110">Phreel</a></p>
+
+Did you take inspiration for the aesthetics of the overlay? If yes, from where and what inspired you the most?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7696512" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Hoaq.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/7696512">Hoaq</a></p>
+
+Saying we didn't would be a blatant lie. Learning from other tournaments (osu! or any other esports) is our go-to solution. There aren't any tournament overlays that really gave us the direction for our overlay.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8266808" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-[Boy]DaLat.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8266808">[Boy]DaLat</a></p>
+
+Speaking of the layout, in fact, I mainly took inspirations from Corsace. For the aesthetic, I'm mostly influenced by the design of Google's Material UI, specifically Android 12's Material You. If you look at the initial prototype design of ResCup overlay, you will notice a heavy resemblance from it. I'm trying to switch to a different style at the moment, which is Brutalism design. Maybe it will appear in my future overlays.
```

Comment:
```suggestion
Speaking of the layout, in fact, I mainly took inspirations from Corsace. For the aesthetic, I'm mostly influenced by the design of Google's Material UI, specifically Android 12's Material You. If you look at the initial prototype design of ResCup overlay, you will notice a heavy resemblance from it. I'm trying to switch to a different style at the moment, which is brutalism. Maybe it will appear in my future overlays.
```